app/app.rb

```diff
@@ -63,6 +64,13 @@ def image
                    .where(id: params[:id], :albums => {:user_id => user_id})
                    .take || halt(404)
 	end
+
+  #
+  # Logging, who'd a thunk
```

Comment:
Please update this method description with what the method does. It's obvious what the method does, but the documentation can be better.